### DISCUSSION TITLE
[BUG] Add the add user to sudo step while installing CS1.6

### DISCUSSION THREAD

**Discussion by jeffdare** [GameServerManagers/LinuxGSM#3394] [Q&A] [answered]
On the install page of CS 1.6 - https://linuxgsm.com/lgsm/csserver/

We need to add that user to the sudo group, else we will not be able to install the dependencies. So please add this step after creating the user

`usermod -aG sudo username`

**Comment by h3o66** [comment]
Not a bug - you can decide if you allow the user to install it or you install it manually via the root user.

**Comment by Paraguanads** [comment]
Doesn't work with `usermod -aG sudo username`. I'm running Ubuntu 20.04 and I'm stucked in this step.

**Comment by h3o66** [comment, marked as answer]
@Paraguanads might be the best for you to move to the discussions or to discord to get help.

**Reply by Paraguanads** [reply]
Thanks a lot! I solved this yesterday with Discord community. Is all about permissions `chmod`
